src/anm/animation/element.js

```diff
@@ -2884,6 +2885,46 @@ Element.getMatrixOf = function(elm, m) {
     return t;
 };
 
+Element.getTranslate = function(elm) {
+    if (elm.parent.layer2Bone) {
+	elm.$bonePath = elm.$bonePath || Element.bonePath(elm);
+        var result = {x: 0; y: 0};
+        var bone = null;
+	for (var li = elm.$bonePath.length; li--;) {
+            bone = elm.$bonePath[li];
+	    result.x += bone.bonelength * Math.cos(bone.bonerotate);
+	    result.y += bone.bonelength * Math.sin(bone.bonerotate); 
+	}
+        elm = bone ? elm.parent.children[bone.$from] : elm;
+        result.x += elm.x;
```

Comment:
Actually these functions could be moved into bone tween(s) code, for example see [rotate-to-path](https://github.com/Animatron/player/blob/develop/src/anm/animation/tween.js#L263) implementation (and I'd personally prefer it this way) .